### PR TITLE
Release notes ydb cli 2 8 0

### PR DESCRIPTION
**Pull request by Gazizonoki**
### Changelog entry 

...

### Changelog category 

* Documentation (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/docs/en/core/changelog-cli.md [hunk 1]
```diff
@@ -2,13 +2,41 @@
 
 # {{ ydb-short-name }} CLI changelog
 
+## Version 2.8.0 {#2-8-0}
+
+Released on January 12, 2024. To update to version **2.8.0**, select the [Downloads](downloads/index.md#ydb-cli) section.
+
+**Features:**
+
+* Added configuration management commands for the cluster `ydb admin config` and `ydb admin volatile-config`.
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:11]: link to documentation?

File: ydb/docs/en/core/changelog-cli.md [hunk 2]
```diff
@@ -2,13 +2,41 @@
...
+
+* Added the `generic` type of requests in the [ydb table query execute](reference/ydb-cli/table-query-execute.md) command, allowing to perform [DDL](https://en.wikipedia.org/wiki/Data_Definition_Language) and [DML](https://en.wikipedia.org/wiki/Data_Manipulation_Language) operations, with arbitrarily-sized results and with support for [MVCC](concepts/mvcc.md). The command uses an experimental API, compatibility is not guaranteed.
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:13]: it's also better to move all these experimental features to the bottom of the feature list

File: ydb/docs/en/core/changelog-cli.md [hunk 3]
```diff
@@ -2,13 +2,41 @@
...
+
+* Added support for loading postgres-types by command [ydb import file csv|tsv|json](reference/ydb-cli/export_import/import-file.md). Only for row tables.
+
+* Added support for directory load from an S3-compatible storage in the [ydb import s3](reference/ydb-cli/export_import/s3_import.md) command. Currently only available on Linux and Mac OS.
+
+* Added support for outputting the result of executing commands [ydb table query execute](reference/ydb-cli/table-query-execute.md), [ydb yql](reference/ydb-cli/yql.md) and [ydb scripting yql](reference/ydb-cli/scripting-yql.md) in [parquet](https://parquet.apache.org/docs/) format.
+
+* In the `ydb table query explain` command, the `--collect-diagnostics` option has been added to collect query diagnostics and save it to a file. The command uses an experimental API, compatibility is not guaranteed.
+
+* In the [ydb workload](reference/ydb-cli/commands/workload/index.md) commands, the `--executer` option has been added, specifying the type of queries used.
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:23]:
```suggestion
* In the [ydb workload](reference/ydb-cli/commands/workload/index.md) commands, the `--executer` option has been added, which allows to specify which type of queries to use.
```

File: ydb/docs/en/core/changelog-cli.md [hunk 4]
```diff
@@ -2,13 +2,41 @@
...
+
+* Added column of median benchmark execution time in the statistics table in the command [ydb workload clickbench](reference/ydb-cli/workload-click-bench.md).
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:25]:
```suggestion
* Added a column with median benchmark execution time in the statistics table of the [ydb workload clickbench](reference/ydb-cli/workload-click-bench.md) command.
```

**Review by blinkov — CHANGES_REQUESTED** [review]

File: ydb/docs/en/core/changelog-cli.md (hunk 1 above)
Comment (Gazizonoki) [reply ydb/docs/en/core/changelog-cli.md:11]: Translation of this documentation is not ready yet.

File: ydb/docs/en/core/changelog-cli.md [hunk 5]
```diff
@@ -2,13 +2,41 @@
...
 # {{ ydb-short-name }} CLI changelog
 
+## Version 2.8.0 {#2-8-0}
+
+Released on January 12, 2024. To update to version **2.8.0**, select the [Downloads](downloads/index.md#ydb-cli) section.
+
+**Features:**
+
+* Added configuration management commands for the cluster `ydb admin config` and `ydb admin volatile-config`.
+
```
Comment (asmyasnikov) [inline ydb/docs/en/core/changelog-cli.md:12]: For what you use empty lines?

**Review by fomichev3000 — APPROVED** [review]
LGTM

**Review by blinkov — APPROVED** [review]

**Pull request merged**